src/scripts/run-build.sh

```diff
@@ -23,7 +23,14 @@ if [[ $os = CYGWIN* || $os = MINGW* || $os = MSYS* ]]; then
 fi
 
 # create buildtool command from parameters
-buildCommand="buildtool ${PARAM_TASKS}"
+buildCommand="buildtool"
+
+if [ -n "$PARAM_TASKS" ]; then
+    buildCommand+=" ${PARAM_TASKS}"
+fi
+if [ -n "$PARAM_BUILD_OPTIONS" ]; then
+    buildCommand+=" ${PARAM_BUILD_OPTIONS}"
+fi
```

Comment:
Probably worth writing a test that uses task args to confirm they are properly passed through to tasks.

Something like:
```yaml
 - matlab/run-build:
       tasks: test("unit", OutputDetail=4)
```